tests/test_bgp_af.rb

```diff
@@ -28,31 +28,254 @@ def setup
     super
     # Disable and enable feature bgp before each test to ensure we
     # are starting with a clean slate for each test.
-    config('no feature bgp', 'feature bgp')
+    if platform == :nexus
+      config('no feature bgp', 'feature bgp')
+    elsif platform == :ios_xr
+      config('no router bgp')
+      config('no route-policy drop_all')
+    end
   end
 
-  def get_bgp_af_cfg(asn, vrf, af)
-    afi, safi = af
-    string =
-      @device.cmd("show run bgp all | sec 'bgp #{asn}' |  sec 'vrf #{vrf}' | " \
-                  "sec 'address-family #{afi} #{safi}' | no-more")
-    string
+  # rubocop:disable Style/WordArray
+  # rubocop:disable Style/ClassVars
+  # rubocop:disable Metrics/LineLength
```

Comment:
* `Style/WordDisable` - Why do you want to disable this?  The matrices look cleaner to me with the word array:

```diff
-    # rubocop:disable Style/WordArray
-    master = [['lax', 'sfo'],
-              ['lax', 'sjc'],
-              ['nyc', 'sfo', 'copy-attributes'],
-              ['sjc', 'nyc', 'copy-attributes']]
+    master = [%w(lax sfo),
+              %w(lax sjc),
+              %w(nyc sfo copy-attributes),
+              %w(sjc nyc copy-attributes)]
```

* `Style/ClassVars` - Do the test variables need to be class variables?  I feel this style doesn't need to be disabled.

* `Metrics/LineLength` - This doesn't need to be disabled.  Instead you can increase the `Max` value allowed, which is set inside `tests/.rubocop.yml`

PSA: WordArray is an autocorrectable offense, which means you can just run `rubocop -a` to change the arrays to word arrays.